cmd/concourse/worker.go

```diff
@@ -50,6 +50,8 @@ type WorkerCommand struct {
 
 	Garden GardenBackend `group:"Garden Configuration" namespace:"garden"`
 
+	ExternalGardenAddress flag.URL `long:"external-garden-address" description:"garden server connection address, when specified will not use embedded garden"`
```

Comment:
Can we rename this to `ExternalGardenURL`? In the other flags, 'Address' typically means IP or IP:Port.

```suggestion
	ExternalGardenURL flag.URL `long:"external-garden-url" description:"API endpoint of an externally managed Garden server to use instead of running one."`
```